src/app/vault/add-edit.component.ts

```diff
@@ -84,4 +89,8 @@ export class AddEditComponent extends BaseAddEditComponent implements OnChanges,
         return (!this.editMode || this.cloneMode) && this.ownershipOptions
             && (this.ownershipOptions.length > 1 || !this.allowPersonal);
     }
+
+    markPasswordAsDirty() {
+        this.form.controls['Login.Password'].markAsDirty();
+      }
```

Comment:
Just a small formatting nit (remove extra indentation, aligning under `markPasswordAsDirty`.
```suggestion
    }
```

Reply:
I've corrected the above issue.